plugins/woocommerce-admin/client/products/fields/variations/variations.scss

```diff
@@ -0,0 +1,39 @@
+.woocommerce-product-variations {
+    min-height: 300px;
```

Comment:
Note that this is temporary and height will be adjusted further in https://github.com/woocommerce/woocommerce/issues/35789.